src/Validators/NameValidator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Portal\Validators;
+
+class NameValidator
+{
+    public static function validateExistsAndLength(string $validateName, int $characterLength)
+    {
+        if (empty($validateName) == false && strlen($validateName) <= $characterLength) {
+            return $validateName;
+        } else {
+            throw new \Exception('An input string does not exist or is too long');
+        }
+    }
+
+    public static function validateLength(string $validateName, int $characterLength)
+    {
+        if ($validateName == '') {
+            return null;
+        } elseif (strlen($validateName) <= $characterLength) {
+            return $validateName;
+        } else {
+            throw new \Exception('An input string does not exist or is too long');
+        }
+    }
+
+    public static function sanitiseString($validateName)
+    {
+        $clean = filter_var($validateData, FILTER_SANITIZE_STRING);
+        $clean = trim($clean);
+        return $clean;
```

Comment:
I think you can change this to:
```suggestion
        return trim(filter_var($validateData, FILTER_SANITIZE_STRING));
```